etl/steps/data/garden/worldbank_wdi/2024-05-20/wdi.meta.yml

```diff
@@ -1469,31 +1468,31 @@ tables:
         title: Primary government expenditures as a proportion of original approved budget (%)
       hd_hci_ovrl:
         title: Human capital index (HCI) (scale 0-1)
-        unit: ""
+        unit: ''
```

Comment:
In this file there are some (I think 12) instances of display name that don't have a `title_public`. Because of this, the grapher step raises a warning (and for good reason, since some of the display names are very short and ambiguous, like "Coal"). In those cases, you can simply add a presentation.title_public (similar to the actual title).

Reply:
Good point. I've fixed them, though I'm not sure what the guidelines for proper titles are. If you have any recommendations, I'm all ears.

Reply:
Given that all other public titles without a display will default to the title, it's fine to simply copy the title.